geode-core/src/main/java/org/apache/geode/internal/tcp/TCPConduit.java

```diff
@@ -945,6 +962,10 @@ public boolean useSSL() {
     return useSSL;
   }
 
+  public boolean useDirectReceiveBuffers() {
+    return !useSSL();
+  }
+
```

Comment:
previous question might be resolved following this accessor... for ssl/tls we use NIO and !ssl we use the directbuffers, cool and with 6 you get eggroll